extensions/Bookalope/index.html

```diff
@@ -1,103 +1,319 @@
+						       class="spectrum-Form-itemLabel spectrum-FieldLabel--left for-checkbox">
+							Full version (paid)
+						</label>
+						<div class="spectrum-Form-itemField">
+							<label class="spectrum-Checkbox" for="input-book-version">
+								<input type="checkbox" name="input-book-version" id="input-book-version"
+								       class="spectrum-Checkbox-input" disabled>
+	                            <span class="spectrum-Checkbox-box">
+					            <svg class="spectrum-Icon spectrum-UIIcon-CheckmarkSmall spectrum-Checkbox-checkmark"
+					                 focusable="false" aria-hidden="true">
+					              <use xlink:href="#icon-CheckmarkSmall"/>
+					            </svg>
+				            </span>
+							</label>
+						</div>
+					</div>
+					<div class="spectrum-Form-item">
+						<label for="input-book-autoclean"
+						       class="spectrum-Form-itemLabel spectrum-FieldLabel--left for-checkbox">Automatic
+							cleanup</label>
+						<div class="spectrum-Form-itemField">
+							<label class="spectrum-Checkbox" for="input-book-autoclean">
+								<input type="checkbox" name="input-book-autoclean" id="input-book-autoclean"
+								       checked="true"
+								       class="spectrum-Checkbox-input" disabled>
+	                            <span class="spectrum-Checkbox-box">
+					            <svg class="spectrum-Icon spectrum-UIIcon-CheckmarkSmall spectrum-Checkbox-checkmark"
+					                 focusable="false" aria-hidden="true">
+					              <use xlink:href="#icon-CheckmarkSmall"/>
+					            </svg>
+				            </span>
+							</label>
+						</div>
+					</div>
+					<div class="spectrum-Form-item">
+						<label for="input-book-highlight-issues"
+						       class="spectrum-Form-itemLabel spectrum-FieldLabel--left for-checkbox">Highlight
+							issues</label>
+						<div class="spectrum-Form-itemField">
+							<label class="spectrum-Checkbox" for="input-book-highlight-issues">
+								<input type="checkbox" name="input-book-highlight-issues"
+								       id="input-book-highlight-issues"
+								       class="spectrum-Checkbox-input" disabled>
+	                            <span class="spectrum-Checkbox-box">
+					            <svg class="spectrum-Icon spectrum-UIIcon-CheckmarkSmall spectrum-Checkbox-checkmark"
+					                 focusable="false" aria-hidden="true">
+					              <use xlink:href="#icon-CheckmarkSmall"/>
+					            </svg>
+				            </span>
+							</label>
+						</div>
+					</div>
+				</div>
+
+
+				<div class="form__nav">
+					<button class="spectrum-Button spectrum-Button--cta" id="button-send">
+						<span class="spectrum-Button-label">Upload and convert</span>
+					</button>
+				</div>
+			</div>
+
+
+			<div id="bookalope-update" class="hidden">
+				<div class="form__nav">
+					<button class="spectrum-Button spectrum-Button--cta" id="button-update">
+						<span class="spectrum-Button-label">Upload from server</span>
```

Comment:
I think I’d like to change the text of the button to `"Refresh from server"`. When clicked, then the Javascript should call the [convert()](https://github.com/jenstroeger/Bookalope-CEP/pull/1/files#diff-976ab6e2d949b3c2ec00dc335f717099L439) function: download the ICML file again and open a new document in InDesign.